src/pages/billing/List.js

```diff
@@ -241,54 +237,61 @@ export const BillingList = ajaxCaller(class BillingList extends Component {
+            style: {
+              textOverflow: 'ellipsis', whiteSpace: 'nowrap',
+              fontSize: '1.25rem', overflowX: 'hidden'
+            }
+          }, [selectedName])
+        ])
       ]),
-      h(PageBox, [
-        div({ style: Style.cardList.toolbarContainer }, [
-          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Billing Projects Management'])
-        ]),
-        div({ style: Style.cardList.cardContainer }, [
-          h(NewBillingProjectCard, {
-            onClick: async () => {
-              try {
-                await Auth.ensureBillingScope()
-                this.setState({ creatingBillingProject: true })
-              } catch (error) {
-                reportError('Failed to grant permissions', 'To create a new billing project, you must allow Terra to view your Google billing account(s).')
-              }
+      div({ style: { display: 'flex', flex: 1, flexWrap: 'wrap' } }, [
+        div({ style: { width: 330, boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)' } }, [
+          div({
+            style: {
+              color: colors.gray[0], backgroundColor: colors.grayBlue[5], fontSize: 16, padding: '1rem 1.5rem',
+              fontWeight: 600, textTransform: 'uppercase', borderBottom: `0.5px solid ${colors.grayBlue[2]}`
             }
-          }),
-          div({ style: { flexGrow: 1 } }, [
-            _.flow(
-              _.filter(({ projectName }) => Utils.textMatch(filter, projectName)),
-              _.map(project => {
-                return h(ProjectCard, {
-                  project, key: `${project.projectName}`
-                })
-              })
-            )(billingProjects)
+          },
+          [
+            'Billing Projects', buttonSecondary({ onClick: () => { this.setState({ creatingBillingProject: true }) } }, [
+              span({
+                style: {
+                  borderRadius: 5, backgroundColor: 'white', padding: '0.5rem',
+                  margin: '1rem', color: colors.green[0], fontSize: 16, fontWeight: 500,
+                  boxShadow: Style.standardShadow
+                }
+              },
+              ['New ', icon('plus-circle', { size: 21 })])
+            ])
           ]),
-          (!isDataLoaded || updating) && spinnerOverlay
+          _.map(project => h(ProjectTabs, {
+            project, key: `${project.projectName}`,
+            isActive: !!selectedName && project.projectName === selectedName
+          }), billingProjects)
         ]),
         creatingBillingProject && h(NewBillingProjectModal, {
           onDismiss: () => this.setState({ creatingBillingProject: false }),
-          onSuccess: () => this.refresh()
-        })
+          onSuccess: () => {
+            this.setState({ creatingBillingProject: false })
+            this.loadProjects()
+          }
+        }),
+        !!selectedName && h(ProjectDetail, { project: _.find({ projectName: selectedName }, billingProjects) }),
```

Comment:
Currently the list of users is not updating when you click between projects. The refresh is happening in `componentDidMount`, but the component is staying mounted so this only gets called once. There are a couple ways to fix this, but the most straightforward for now is to add `selectedName` as a `key` to this element, which will force it to remount each time it changes.